app/models/movie.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 class Movie < ApplicationRecord
+  has_many: screenings, dependent: :destroy
+  validates_associated :screenings
```

Comment:
Any particular reason why do you use `validates_associated`?

Reply:
When I double thought on that, I see there is no sens in this case use it.